src/chrome/content/rules/GOOD.is.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+	Related Ruleset:
+		goodstatic.com.xml
+-->
+<ruleset name="GOOD.is">
+	<target host="good.is" />
+	<target host="www.good.is" />
```

Comment:
You may add it as a test url, but I think we should avoid wildcrads if possible.